ar_exercise/exercise.rb

```diff
@@ -54,18 +73,40 @@ class Apartment < ActiveRecord::Base
 # Create at least 9 new tenants and save them to the DB. (Make sure they belong to an apartment)
 # Note: you'll use this little bit of code as a `seeds.rb` file later on.
 
+mich_house = Apartment.create(address: "1227 Michigan Pass", monthly_rent: 3500, sqft: 2300, num_beds:6, num_baths:4)
+gardener_st = Apartment.create(address: "73 Gardener Avenue" monthly_rent: 2000, sqft: 1800, num_beds:3, num_baths:3)
+beacon_lane = Apartment.create(address: "4000 Beacon Lane" monthly_rent: 2500, sqft: 1500, num_beds:2, num_baths:1)
+
+mathew = Tenant.create(name: "Mathew	Cross", age:19, gender:"Male", apartment:beacon_lane)
+joella = Tenant.create(name: "Joella	Salter", age:24, gender:"Female", apartment:gardener_st)
+malcolm = Tenant.create(name: "Malcolm	Bryant" age: 45, gender: "Male", apartment: mich_house)
+erin = Tenant.create(name: "Erin	Patrick", age: 31, gender:"Female", apartment: mich_house)
+jeremiah = Tenant.create(name: "Jeremiah	Malone" age: 90, gender: "Male", apartment: mich_house)
+claudia = Tenant.create(name: "Claudia	Howell", age: 30, gender: "Female", apartment: gardener_st)
+damon = Tenant.create(name: "Damon	Fisher", age: 20, gender: "Male", apartment: beacon_lane)
+holly = Tenant.create(name: "Holly	Fields", age: 34, gender: "Female", apartment: mich_house)
+rufus = Tenant.create(name: "Rufus	Dunn", age: 55, gender: "Male" apartment: mich_house)
+
```

Comment:
You've hard coded the apartment_id, which generally a bad practice. Fortunately ActiveRecord is super smart and will add the appropriate apartment_id when we create a tenant on an apartment
```rb
mitch_house.tenants.create("a bunch of key value pairs, not including apartment_id")
```